css/main.scss

```diff
@@ -49,5 +47,6 @@ $on-laptop:        800px;
 @import
         "base",
         "layout",
-        "syntax-highlighting"
+        "syntax-highlighting",
+        "kakapo"
```

Comment:
maybe should not be always imported since it's not used by other pages and is specific to documentation